mypy/stubtest.py

```diff
@@ -1044,8 +1036,13 @@ def verify_paramspecexpr(
         return
 
 
-def _verify_readonly_property(stub: nodes.Decorator, runtime: Any) -> Iterator[str]:
+def _verify_property_decorator(stub: nodes.Decorator, runtime: Any) -> Iterator[str]:
```

Comment:
I'd still prefer a name that makes clear that this function *won't* be called for a property if the property has a setter in the stub

Reply:
It won't be `Decorator` in this case 🤔 
Any suggestions?

Reply:
I'm not sure I see the need for `_verify_abstract_status` to be in a separate function to the other miscellaneous property-verification logic. I would keep the function name as it is, and just inline the additional logic into the existing function :)